test/specs/elements/Button/Button-test.js

```diff
@@ -126,4 +126,13 @@ describe('Button', () => {
       wrapper.children().at(1).should.match('.ui.label')
     })
   })
+
+  describe('labeled', () => {
+    it('is before the button and icon="plus" when labeled="left"', () => {
+      const wrapper = shallow(<Button labeled='left' icon='plus' />)
+      wrapper.should.have.tagName('button')
+      wrapper.should.have.className('labeled')
+      wrapper.should.have.exactly(1).descendants('Icon')
+    })
```

Comment:
Thanks for the test, let's simplify it a bit.  It is best to write tests that assert one thing as they are stronger and more maintainable.  In this case, most of the assertions made here are covered by other existing tests, all we should need to assert is that a `labeled` button renders a `button` element:

```
it('renders as a div', () => {
  shallow(<Button labeled />)
    .should.have.tagName('button')
})
```